README.md

```diff
@@ -41,10 +41,10 @@ ___
 ### Dependencies
 
 PebblesDB requires `libsnappy` and `libtool`. To install on Linux, please use
-`sudo apt-get install libsnappy-dev libtool`.
+`sudo apt-get install libsnappy-dev libtool`. For MacOSX, use `brew install snappy` and instead of `ldconfig`, use `update_dyld_shared_cache`.
+
+PebblesDB was built and compiled with g++-5, so it might not compile with later versions of g++ now. 
```

Comment:
Do we want to mention it will work on g++-4?

Reply:
We can say it'll work on g++ v4. I haven't tested on Linux. I'll do that

Reply:
Tested on Linux. Compiles fine. Test hangs at bloom filter. When commented out, others pass.
Thinking more about the previous answer, we haven't tested on g++4 on linux. Though g++ version on Mac is 4, it might be using clang. So we might not have handled all dependencies for g++4.

Reply:
Can you test with g++4 on Linux and commit a patch to the README? We'll make it part of this PR.

Reply:
Okay, will do that.

Reply:
Works fine with g++4.7 and g++4.9 as well. Tested on the server.

Reply:
great! Can you commit the change to the README?